ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh

```diff
@@ -33,28 +39,29 @@ cp /tmp/ssh-publickey ~/.ssh/id_rsa.pub
 cp /tmp/ssh-privatekey ~/.ssh/id_rsa
 chmod 0400 ~/.ssh/id_rsa*
 
-sudo subscription-manager register \
-  --org="$(cat /var/run/rhsm/subscription-manager-org)" \
-  --activationkey="$(cat /var/run/rhsm/subscription-manager-act-key)"
+# Set up the pull secret in the expected location
+export PULL_SECRET="\${HOME}/.pull-secret.json"
+cp /tmp/pull-secret "\${PULL_SECRET}"
 
 cd ~/microshift
-# Get firewalld and repos in place. Use scripts to get the right repos
-# for each branch.
-./scripts/devenv-builder/configure-vm.sh --no-build-deps --force-firewall /tmp/pull-secret
-./scripts/image-builder/configure.sh
-./scripts/image-builder/build.sh -pull_secret_file /tmp/pull-secret -microshift_rpms ~/rpms -authorized_keys_file /tmp/ssh-publickey -open_firewall_ports 6443:tcp
+
+./test/bin/ci_phase_iso_build.sh
+
+./scripts/image-builder/build.sh -pull_secret_file "\${PULL_SECRET}" -microshift_rpms ~/rpms -authorized_keys_file ~/.ssh/id_rsa.pub -open_firewall_ports 6443:tcp
```

Comment:
Just a note for the future, as it is part of https://github.com/openshift/release/pull/40429, the rpms.tar we are copying into the metal host are the same rpms that its going to build. The -microshift_rpms option could be removed so that it takes the ones it just built.

Reply:
Maybe we can do that in the PR where you remove the build step from this pipeline?